foyer/tests/utils.py

```diff
@@ -3,8 +3,7 @@
 import numpy as np
 
 
-
-def atomtype(structure, forcefield):
+def atomtype(structure, forcefield, non_atomistic=False):
```

Comment:
Correct.  In `Forcefield` we create new 'elements' for the non-atomistic particles, but the tests are taking elements straight from the MOL2 reader.